src/modules/sensors/vehicle_magnetometer/VehicleMagnetometer.hpp

```diff
@@ -134,6 +144,7 @@ class VehicleMagnetometer : public ModuleParams, public px4::ScheduledWorkItem
 
 	DEFINE_PARAMETERS(
 		(ParamInt<px4::params::CAL_MAG_COMP_TYP>) _param_mag_comp_typ,
+		(ParamInt<px4::params::SENS_MAG_MODE>) _param_sens_mag_mode,
```

Comment:
```suggestion
		(ParamBool<px4::params::SENS_MAG_MODE>) _param_sens_mag_mode,
```

Reply:
The reason it's SENS_x_MODE and not something about enable voting is that I wanted to preserve the possibility of actually having 3 identical synchronized sensors to median select. I'll change it to bool for now.